pkg/compute/publish.go

```diff
@@ -0,0 +1,98 @@
+package compute
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/text"
+)
+
+// PublishCommand produces and deploys an artifact from files on the local disk.
+type PublishCommand struct {
+	common.Base
+	manifest manifest.Data
+	build    *BuildCommand
+	deploy   *DeployCommand
+
+	// Deploy fields
+	path    common.OptionalString
+	version common.OptionalInt
+
+	// Build fields
+	name       common.OptionalString
+	lang       common.OptionalString
+	includeSrc common.OptionalBool
+	force      common.OptionalBool
+}
+
+// NewPublishCommand returns a usable command registered under the parent.
+func NewPublishCommand(parent common.Registerer, globals *config.Data, build *BuildCommand, deploy *DeployCommand) *PublishCommand {
+	var c PublishCommand
+	c.Globals = globals
+	c.manifest.File.SetOutput(c.Globals.Output)
+	c.manifest.File.Read(manifest.Filename)
+	c.build = build
+	c.deploy = deploy
+	c.CmdClause = parent.Command("publish", "Composite of \"build\", then \"deploy\"")
```

Comment:
'Composite' seems a bit of a complicated way to describe it, and requires the user to go look up two other things.  Maybe "Build a Compute@Edge package and deploy it to a Fastly service" ?

Reply:
👍🏻 to @triblondon's suggestion.

Reply:
This is why I was happy GitHub chose you to review 🙂 because my description really sucked.